docs/configuration.md

```diff
@@ -1751,6 +1751,7 @@ Apart from these, the following properties are also available, and may be useful
     <code>spark.dynamicAllocation.minExecutors</code>,
     <code>spark.dynamicAllocation.maxExecutors</code>, and
     <code>spark.dynamicAllocation.initialExecutors</code>
+    <code>spark.dynamicAllocation.fullExecutorAllocationDivisor</code>
```

Comment:
needs changed to executorAllocationRatio

Reply:
Done, missed that one, sorry :-)